vtk-cdat/meta.yaml.in

```diff
@@ -3,7 +3,7 @@ package:
     version: 7.1.0.@VERSION@
 
 source:
-    git_rev: master
+    git_rev: f03521b50c4b630d4dcf23d237ef2d9470581306
```

Comment:
@doutriaux1 I have seen that. This is very nice, thanks! In the same time I don't think we should update the VTK SHA automatically just because VTK master was updated. If we do, we introduce an additional variability - we probably update vcs and VTK got updated just because we rebuild. I think we should only update VTK when we need new features or bug fixes from it.